lua/lang/r.lua

```diff
@@ -0,0 +1,55 @@
+local M = {}
+
+M.config = function()
+  -- R -e 'install.packages("formatR",repos = "http://cran.us.r-project.org")'
+  -- R -e 'install.packages("readr",repos = "http://cran.us.r-project.org")'
+  O.lang.r = {
+    formatter = {
+      exe = "R",
+      args = {
+        "--slave",
+        "--no-restore",
+        "--no-save",
+        '-e "formatR::tidy_source(text=readr::read_file(file(\\"stdin\\")), arrow=FALSE)"',
+      },
+    },
+  }
+end
+
+M.format = function()
+  O.formatters.filetype["r"] = {
+    function()
+      return {
+        exe = O.lang.r.formatter.exe,
+        args = O.lang.r.formatter.args,
+        stdin = not (O.lang.r.formatter.stdin ~= nil),
```

Comment:
This is equivalent to stdin = (O.lang.r.formatter.stdin == nil), right ?
So stdin == true if O.lang.r.formatter.stdin == nil, is this right ?

Reply:
🤦  yeah you are right, gotta fix them all in a new PR probably 😭